Sofa/GUI/Qt/src/sofa/gui/qt/viewer/qt/QtViewer.cpp

```diff
@@ -932,8 +932,8 @@ void QtViewer::calcProjection(int width, int height)
         currentCamera->setBoundingBox(vparams->sceneBBox().minBBox(), vparams->sceneBBox().maxBBox());
     }
     currentCamera->computeZ();
-    currentCamera->p_widthViewport.setValue(width);
-    currentCamera->p_heightViewport.setValue(height);
+    currentCamera->p_widthViewport.setValue(sofa::Size(width));
```

Comment:
as we are now using user defined litteral for _sreal... maybe we could have others for _size and _index ?

Reply:
Sure, but litterals applies to constants, which is not the case here.